src/core/MathUtils.cpp

```diff
@@ -330,11 +330,13 @@ OCIO_NAMESPACE_ENTER
     namespace
     {
     
+#if defined(OpenColorIO_Static) || defined(OCIO_UNIT_TEST)
```

Comment:
> defined(OpenColorIO_Static)

Why did you add the 'static lib.' condition (i.e. unit test condition seems enough)?

Reply:
It looks like that did work. During my testing earlier I thought that I had tried that and that it complained that the function was not defined for the Static lib. Looks like I had something else wrong. I'll fix that.